.appveyor.yml

```diff
@@ -20,34 +20,106 @@ install:
-  - '"C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe" "C:\projects\mlpack\build\mlpack.sln" /m /verbosity:minimal /nologo /p:BuildInParallel=true /p:Configuration=Release;Platform=x64'
+  - >
+    cmake -G "Visual Studio 14 2015 Win64"
+    -DBLAS_LIBRARY:FILEPATH="%APPVEYOR_BUILD_FOLDER%/OpenBLAS.0.2.14.1/lib/native/lib/x64/libopenblas.dll.a"
+    -DLAPACK_LIBRARY:FILEPATH="%APPVEYOR_BUILD_FOLDER%/OpenBLAS.0.2.14.1/lib/native/lib/x64/libopenblas.dll.a"
+    -DARMADILLO_INCLUDE_DIR="C:/projects/mlpack/armadillo-7.800.2/include"
+    -DARMADILLO_LIBRARY:FILEPATH="C:\projects\mlpack\armadillo-7.800.2\build\Debug\armadillo.lib"
+    -DBOOST_INCLUDEDIR:PATH="C:\projects\mlpack\boost.1.60.0.0\lib\native\include"
+    -DBOOST_LIBRARYDIR:PATH="C:\projects\mlpack\boost_libs"
+    -DDEBUG=OFF
+    -DPROFILE=OFF
+    -DBUILD_PYTHON_BINDINGS=OFF
+    -DCMAKE_BUILD_TYPE=Release ..
+  - >
+    "C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe"
+    "C:\projects\mlpack\build\mlpack.sln"
+    /m /verbosity:minimal /nologo /p:BuildInParallel=true /p:Configuration=Release;Platform=x64
   
-  # Zip Artifacts
+  # Zip Artifacts.
   - 7z a mlpack-windows-no-libs.zip "%APPVEYOR_BUILD_FOLDER%\build\Release\*.exe"
-  - 7z a mlpack-windows.zip "%APPVEYOR_BUILD_FOLDER%\build\Release\*.*" "%APPVEYOR_BUILD_FOLDER%/OpenBLAS.0.2.14.1/lib/native/lib/x64/*.*"
+  - >
+    7z a mlpack-windows.zip
+    "%APPVEYOR_BUILD_FOLDER%\build\Release\*.*"
+    "%APPVEYOR_BUILD_FOLDER%/OpenBLAS.0.2.14.1/lib/native/lib/x64/*.*"
   
-  # Pulling documentation for the installer
-  - ps: try{(new-object net.webclient).DownloadFile('http://ci.mlpack.org/job/mlpack%20-%20doxygen%20build/lastSuccessfulBuild/artifact/build/doc/html/*zip*/html.zip', 'C:\projects\mlpack\dist\win-installer\jenkinsdoc.zip')}catch{Write-Output "Unable to pull jenkins doc, skipping!"}
-  - ps: try{(Add-Type -AssemblyName System.IO.Compression.FileSystem); [System.IO.Compression.ZipFile]::ExtractToDirectory('C:\projects\mlpack\dist\win-installer\jenkinsdoc.zip', 'C:\projects\mlpack\dist\win-installer\staging\doc')}catch{Write-Output "Unable to add doc to installer, skipping!"}
+  # Pulling documentation for the installer.
+  - ps: >
+      try{(new-object net.webclient).DownloadFile('http://ci.mlpack.org/job/mlpack%20-%20doxygen%20build/lastSuccessfulBuild/artifact/build/doc/html/*zip*/html.zip',
+      'C:\projects\mlpack\dist\win-installer\jenkinsdoc.zip')}catch{Write-Output "Unable to pull jenkins doc, skipping!"}
+  - ps: >
+      try{(Add-Type -AssemblyName System.IO.Compression.FileSystem);
+      [System.IO.Compression.ZipFile]::ExtractToDirectory('C:\projects\mlpack\dist\win-installer\jenkinsdoc.zip',
+      'C:\projects\mlpack\dist\win-installer\staging\doc')}catch{Write-Output "Unable to add doc to installer, skipping!"}
```

Comment:
It would be really nice if we could wrap these to under 80 characters, but given the long URLs I think it may not be possible.  Nice to know about the `>` character though for sure. :+1: